setup.py

```diff
@@ -41,6 +41,7 @@
     packages=find_packages(),
     install_requires=[
         "requests"
+        "six"
```

Comment:
@ronthalanki-wework : I think install requirements are comma (`,`) separated. Did you have any issues installing dependencies locally?

Reply:
yep, i needed to add a comma

Reply:
Great!